runtime/cmake/J9vmTargetSupport.cmake

```diff
@@ -22,13 +22,54 @@
 
 include(OmrTargetSupport)
 
+# Adds copyright metadata on windows. On other platforms its a no-op.
+function(j9vm_add_copyright_data target_name)
+	omr_assert(FATAL_ERROR TEST TARGET ${target_name} MESSAGE "Expected a target name, got ${target_name}")
+
+	if(NOT OMR_OS_WINDOWS)
+		return()
+	endif()
+
+	get_target_property(target_type "${target_name}" TYPE)
+	if(NOT target_type STREQUAL "SHARED_LIBRARY")
+		return()
+	endif()
+
+	if(NOT OPENJDK_VERSION_NUMBER_FOUR_POSITIONS)
+		set(VERSION_STRING "0.0.0.0")
```

Comment:
Should this be an assert instead?

Reply:
Probably, yes. This code was written before the openjdk code which sets the variable was merged